mkdocs_monorepo_plugin/plugin.py

```diff
@@ -26,10 +26,15 @@ def __init__(self):
         self.files_source_dir = {}
 
     def on_config(self, config):
+        """Initialize MonorepoPlugin and return config of aggregated docs
+        folder."""
         # If no 'nav' defined, we don't need to run.
         if not config.get('nav'):
             return config
 
+        # setting originalDocsDir means that on_config has been run
+        self.originalDocsDir = config['docs_dir']
```

Comment:
nit: i think this makes more sense in its original block, where we "update the docs_dir with our temporary one"

Reply:
true, but i think that makes the general flow of the logic less obvious.